maps-utils-v3-ktx/src/main/java/com/google/maps/android/ktx/utils/LatLng.kt

```diff
@@ -0,0 +1,215 @@
+/*
```

Comment:
@arriolac This doesn't look right to me (Kotlin classes added in a docs PR) - would you be able to take a look?

Reply:
Will take a look 🤔